boards/shields/st_b_lcd40_dsi1_mb1166/boards/stm32h747i_disco_m7.overlay

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2023 bytes at work AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/display/stm32_ltdc.h>
+#include <dt-bindings/mipi_dsi/mipi_dsi.h>
+#include <st/h7/stm32h747xihx-pinctrl.dtsi>
+
+/ {
+	chosen {
+		zephyr,display = &ltdc;
+	};
+};
+
+&ltdc {
+	status = "okay";
+	ext-sdram = <&sdram2>;
+	pinctrl-names = "default";
+	pinctrl-0 = <>; /* use DSI */
+
+	/* orisetech, otm8009a */
+	width = <800>;
+	height = <480>;
+	hsync-pol = <STM32_LTDC_HSPOL_ACTIVE_LOW>;
+	vsync-pol = <STM32_LTDC_VSPOL_ACTIVE_LOW>;
+	de-pol = <STM32_LTDC_DEPOL_ACTIVE_LOW>;
+	pclk-pol = <STM32_LTDC_PCPOL_ACTIVE_LOW>;
+	hsync-duration = <2>;
+	vsync-duration = <1>;
+	hbp-duration = <34>;
+	vbp-duration = <15>;
+	hfp-duration = <34>;
+	vfp-duration = <16>;
+
+	def-back-color-red = <0>;
+	def-back-color-green = <0>;
+	def-back-color-blue = <0>;
+};
+
+/* ltdc uses pll3_r as pixel clock */
+&pll3 {
+	status = "okay";
+	clocks = <&clk_hse>;
+	div-m = <5>;
+	mul-n = <132>;
+	div-p = <2>;
+	div-q = <2>;
+	div-r = <24>; /* 27.5 MHz */
+};
+
+&mipi_dsi {
+	status = "okay";
+
+	pll-ndiv = <100>;
+	pll-idf = <5>;
+	pll-odf = <0>;
+
+	vs-active-high;
+	hs-active-high;
+	de-active-high;
+};
+
+&otm8009a {
+	reset-gpios = <&gpiog 3 GPIO_ACTIVE_HIGH>;
+	bl-gpios = <&gpioj 12 GPIO_ACTIVE_HIGH>;
+	data-lanes = <2>;
+	pixel-format = <MIPI_DSI_PIXFMT_RGB888>;
+	rotation = <90>;
+};
```

Comment:
I'm not familiar with the nexus connector concept.
I would appreciate some help.

Reply:
Point would be to define or use an existing connector (see https://docs.zephyrproject.org/3.2.0/hardware/porting/shields.html#gpio-nexus-nodes), which would allow to make shield really portable as it would be totally board pin agnostic.